src/web/nextui/src/app/components/InfoModal.tsx

```diff
@@ -0,0 +1,61 @@
+import BugReportIcon from '@mui/icons-material/BugReport';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import ForumIcon from '@mui/icons-material/Forum';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import MenuBookIcon from '@mui/icons-material/MenuBook';
+import { Modal } from '@mui/material';
+import Link from 'next/link';
+import './InfoModal.css';
+
+export function InfoModal({ open, onClose }: { open: boolean; onClose: () => void }) {
+  return (
+    <Modal open={open} onClose={onClose} className="info-modal">
+      <div className="info-modal-content">
```

Comment:
Looks nice overall, however we should prefer the semantic tags provided by MUI like `<Box>` and `<Typography>`.  This makes sure that the styles are consistent throughout the app, and saves you from writing a bunch of CSS.